test/attributes-to-props.test.js

```diff
@@ -336,23 +335,3 @@ describe('attributesToProps with custom attribute', () => {
     `);
   });
 });
-
-describe('utilities.PRESERVE_CUSTOM_ATTRIBUTES=false', () => {
-  const { PRESERVE_CUSTOM_ATTRIBUTES } = utilities;
-  const emptyProps = {};
-
-  beforeAll(() => {
-    utilities.PRESERVE_CUSTOM_ATTRIBUTES = false;
-  });
-
-  afterAll(() => {
-    utilities.PRESERVE_CUSTOM_ATTRIBUTES = PRESERVE_CUSTOM_ATTRIBUTES;
-  });
-
-  it('omits unknown attributes', () => {
-    const attributes = {
-      unknownAttribute: 'someValue'
-    };
-    expect(attributesToProps(attributes)).toEqual(emptyProps);
-  });
-});
```

Comment:
Since it always returns `true`, I don't think we need to force it to be false to run the test...! 🙏